securesystemslib/interface.py

```diff
@@ -69,6 +69,8 @@
 # Supported key types.
 SUPPORTED_KEY_TYPES = ['rsa', 'ed25519']
 
+# Private keys are created with read and write permissions for the user only
+PRIVATE_KEY_MODE = 0o600
```

Comment:
I know some people prefer the octal form so I don't make any demands, just note that this could be 
`PRIVATE_KEY_MODE = stat.S_IRUSR | stat.S_IWUSR`

Reply:
Yes, let's use the constants.

Reply:
Used!